src/main.rs

```diff
@@ -1,28 +1,25 @@
 #![windows_subsystem = "windows"]
 
-use anyhow::Result;
 use clap::Parser;
 
+mod cli;
 mod gui;
 mod installer;
 
-const ICON: &'static [u8] = include_bytes!("../quilt.png");
+const ICON: &[u8] = include_bytes!("../quilt.png");
 
-#[derive(Default, Parser)]
-#[clap(about, version)]
-pub struct Args {
-    /// Start the installer in no-gui mode
-    #[clap(long)]
-    no_gui: bool
-}
-
-fn main() -> Result<()> {
-    let args = Args::parse();
+fn main() -> anyhow::Result<()> {
+    let args = cli::Args::parse();
 
-    if args.no_gui {
-        println!("No gui mode")
+    if let Some(subcommand) = args.subcommand {
+        match subcommand {
+            cli::SubCommands::Client => println!("Installing client..."),
+            cli::SubCommands::Server => println!("Installing server..."),
+        }
+        std::thread::sleep(std::time::Duration::from_secs(2));
+        println!("Just kidding, we haven't implemented the CLI yet :)");
```

Comment:
While jokes are nice, yeah, I prefer having something less "haha i tricked you, user!" for this
```suggestion
        println!("The CLI hasn't been implemented yet!");
```